themes/default/content/blog/pulumi-release-notes-66/index.md

```diff
@@ -0,0 +1,113 @@
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - meagan-cojocar
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+---
+
+Over the holidays we have been releasing new features and improvements. Read on to learn about what's new this release!
+
+- Cloud Providers and Packages
+  - [New `Command` package]({{<relref "/blog/pulumi-release-notes-66#new-command-package">}})
+  - [Support pulumi import for Kubernetes CRDs]({{<relref "/blog/pulumi-release-notes-m66#support-pulumi-import-for-kubernetes-crds">}})
+  - [Various improvements to Helm Release]({{<relref "/blog/pulumi-release-notes-m66#various-improvements-to-helm-release">}})
+- Pulumi CLI and core technologies
+  - [Support using native ES modules as Pulumi scripts]({{<relref "/blog/pulumi-release-notes-m66#support-using-native-es-modules-as-pulumi-scripts">}})
+  - [Support packages with plugins hosted in any third-party location]({{<relref "/blog/pulumi-release-notes-m66#support-packages-with-plugins-hosted-in-any-third-party-location">}})
+  - [State locking default enabled]({{<relref "/blog/pulumi-release-notes-m66#state-locking-default-enabled">}})
+- Pulumi Service & Pulumi.com
+  - [Third-party audit log tracking for secrets decryption]({{<relref "/blog/pulumi-release-notes-m66#third-party-audit-log-tracking-for-secrets-decryption">}})
+
+<!--more-->
+
+## Cloud Providers and Packages
+
+### New `Command` package
+
+As part of creating or updating infrastructure, it is often necessary to run scripts and/or commands. In order to improve this experience we released a new [Pulumi Command package]({{<relref "/registry/packages/command">}}) in the Pulumi registry which enables users to run scripts locally or remotely on a target VM as part of the Pulumi resource lifecycle.
+
+This new package is supported in all Pulumi languages.
+
+The Command package supports quite a few common patterns involving local and remote scripts execution, such as:
+- [A simple local resource (random)]({{<relref "/registry/packages/command/#a-simple-local-resource-random">}})
+- [Remote provisioning of an EC2 instance]({{<relref "/registry/packages/command/#remote-provisioning-of-an-ec2-instance">}})
+- [Invoking a Lambda during a Pulumi deployment]({{<relref "/registry/packages/command/#invoking-a-lambda-during-pulumi-deployment">}})
+- [Using local.Command with curl to manage an external REST API]({{<relref "/registry/packages/command/#invoking-a-lambda-during-pulumi-deployment">}})
+- [Graceful cleanup of workloads in a Kubernetes cluster]({{<relref "/registry/packages/command/#graceful-cleanup-of-workloads-in-a-kubernetes-cluster">}})
```

Comment:
```suggestion
- [A simple local resource (random)]({{<relref "/registry/packages/command#a-simple-local-resource-random">}})
- [Remote provisioning of an EC2 instance]({{<relref "/registry/packages/command#remote-provisioning-of-an-ec2-instance">}})
- [Invoking a Lambda during a Pulumi deployment]({{<relref "/registry/packages/command#invoking-a-lambda-during-pulumi-deployment">}})
- [Using local.Command with curl to manage an external REST API]({{<relref "/registry/packages/command#invoking-a-lambda-during-pulumi-deployment">}})
- [Graceful cleanup of workloads in a Kubernetes cluster]({{<relref "/registry/packages/command#graceful-cleanup-of-workloads-in-a-kubernetes-cluster">}})
```

Reply:
@cnunciato is this something we can lint for?

Reply:
Not with anything we have now, unfortunately. There's a handful of other things we'll need to add along this same line, though (like checking for root-relative paths, etc.), so it's definitely on my radar. Probably something we'll need to write at least partly ourselves, though.

Reply:
Filed this to track it: https://github.com/pulumi/pulumi-hugo/issues/1003

Reply:
okay, would love to figure out if we can prioritize that kind of work in the future if possible to make our contribution and review process better.

Reply:
> would love to figure out if we can prioritize that kind of work

Yep, it's in my queue, so it'll get looked at. :)